build.sbt

```diff
@@ -20,14 +18,18 @@ val commonDependencies = Seq(
   "ch.qos.logback" % "logback-classic" % "1.5.18"
 )
 lazy val commonSettings = Seq(
-  scalaVersion := "2.13.16",
-  scalacOptions ++= Seq(
-    "-deprecation",
-    "-feature",
-    "-unchecked",
-    "-release:11",
-    "-Xfatal-warnings"
-  ),
+  scalaVersion := "3.3.6",
+  scalacOptions ++= {
```

Comment:
Something like this avoids having to cross reference the common options on each version path:

```scala
scalacOptions ++= Seq(
  "-feature",
  "-release:11"
) ++ (CrossVersion.partialVersion(scalaVersion.value) match {
  case Some((2, 13)) => Seq("-Xfatal-warnings")
  case Some((3, _))  => Seq("-Werror")
  case _             => Seq.empty
}),
```